lib/jekyll/readers/layout_reader.rb

```diff
@@ -19,6 +19,12 @@ def read
           Layout.new(site, theme_layout_directory, layout_file)
       end
 
+      if @layouts.empty?
+        Jekyll.logger.warn "", "No layouts detected. Ensure that your layouts directory is at"
+        Jekyll.logger.warn "", "the root of your source directory or at the root of your theme-gem"
+        Jekyll.logger.warn "", "and that it is not empty."
```

Comment:
A static HTML site is a valid Jekyll site.
This has always been a feature of Jekyll, and adding a warning here seems counter to that policy 👎🏼